cmd/litcli/sessions.go

```diff
@@ -73,7 +73,11 @@ var addSessionCommand = cli.Command{
 				"this flag will only be used if the 'type' " +
 				"flag is set to 'custom'. This flag can be " +
 				"specified multiple times if multiple URIs " +
-				"should be included",
+				"should be included. Note that a wild-card " +
+				"URI can also be specified to include all " +
+				"URIs of a specific server, for example: " +
+				"'--uri=\"/poolrpc.Trader/*\"' will include " +
```

Comment:
nit: this pattern technically isn't a valid regex but a "glob" (or just pattern). But maybe we actually want it to be a full regex so we don't need to do the matching/replacing below? Then I could also select all `lnrpc` permissions with `/lnrpc\..*`?

And does it make sense to print the matched URIs to the console just to show the user what permissions they get in the session?

Reply:
> But maybe we actually want it to be a full regex

Great idea!

> And does it make sense to print the matched URIs to the console just to show the user what permissions they get in the session?

It does currently print them out to the console since the URIs are listed in the macaroon recipe which is printed out. Do you mean that perhaps we should add like a "here are the perms you selected, are you sure you want to continue [y/n] type thing?"

Reply:
> But maybe we actually want it to be a full regex

Done!